net/qpid-proton/Portfile

```diff
@@ -58,3 +63,16 @@ variant swig description {With built-in support for SWIG so the bindings can be
                             port:swig-go \
                             port:swig-ruby
 }
+
+post-build {
+    set scriptname   runtests.sh
+
+    set script [open "${worksrcpath}/../build/${scriptname}" w 0644]
+    puts ${script} "#!/bin/bash"
+    puts ${script} "source ./config.sh"
+    puts ${script} "make test"
+    close ${script}
```

Comment:
Wouldn't it be simpler to store the finished script in the files directory and copy it to the right place, rather than write code to write the file line by line?
```
    xinstall ${filespath}/runtests.sh ${build.dir}
```